crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -551,7 +553,15 @@ where
         start: BlockNumber,
         count: u64,
     ) -> EngineApiResult<ExecutionPayloadBodiesV1> {
-        self.get_payload_bodies_by_range_with(start, count, convert_to_payload_body_v1).await
+        self.get_payload_bodies_by_range_with(start, count, |block| {
+            let transactions =
+                block.body().transactions().iter().map(|tx| tx.encoded_2718().into());
```

Comment:
there should be a helper fn for this:

https://github.com/paradigmxyz/reth/blob/440e6695b5d6e57f47ff73ba54e944e47b5d0be9/crates/primitives-traits/src/block/body.rs#L135-L135